third_party/txt/src/skia/paragraph_skia.cc

```diff
@@ -45,27 +45,25 @@ txt::FontStyle GetTxtFontStyle(SkFontStyle::Slant font_slant) {
 
 class DisplayListParagraphPainter : public skt::ParagraphPainter {
  public:
+  //----------------------------------------------------------------------------
+  /// @brief      Creates a |skt::ParagraphPainter| that draws to DisplayList.
+  ///
+  /// @param      builder  The display list builder.
+  /// @param[in]  dlPaints The paints referenced by ID in the `drawX` methods.
+  /// @param[in]  drawPathEffectNaively  If true, draw path effects naively by
+  ///                                    drawing multiple lines instead of
+  ///                                    providing a path effect to the paint.
+  ///
+  /// @note       Impeller does not support path effects, so this is a stop-gap
```

Comment:
This isn't really a stop gap though, its how we'd ultimately end up implementing the dash path effect anyway.

Reply:
Yeah good point, I more meant, this isn't going to be a runtime flag. I'll make an edit sometime tomorrow AM :)

Thanks for your help!

Reply:
Done.